controllers/datadogagent/systemprobe.go

```diff
@@ -93,6 +95,8 @@ runtime_security_config:
     dir: %s
   syscall_monitor:
     enabled: %v
+remote_tagger: %v
```

Comment:
`remote_tagger` should be inside `runtime_security_config` section

Reply:
The configuration know is `remote_tagger_enabled`

Reply:
thanks for catching this error.